backend/src/main/resources/application.properties

```diff
@@ -1,9 +1,6 @@
 spring.datasource.url=jdbc:postgresql://localhost:5432/restaurants_db
```

Comment:
this file should look like this: 
```
spring.datasource.url=jdbc:postgresql://localhost:5432/{app_name}
spring.datasource.username=
spring.datasource.password=
spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
spring.jpa.hibernate.ddl-auto = create

server.port=
```

Reply:
Fixed in 0cd4debab433260ab6a8267aa0b72b31db581edb